bower.json

```diff
@@ -13,6 +13,7 @@
     "o-grid": "^4.3.1",
     "fontfaceobserver": "^2.0.9",
     "o-icons": ">=4.4.2 <6",
-    "o-spacing": "^2.0.0"
+    "o-spacing": "^2.0.0",
+    "o-normalise": "^1.7.4"
```

Comment:
Shall we add this to MIGRATION.md in this PR or a future PR?

Reply:
A future. It's going to be "check compatibility with all dependencies, there are new ones and they have _all_ upgraded". 😬